frontend/package.json

```diff
@@ -85,6 +86,7 @@
     "react-tagsinput": "3.19.x",
     "react-transition-group": "2.3.x",
     "react-virtualized": "9.x",
+    "reactabular-dnd": "8.x",
```

Comment:
Agreed, declaring an explicit dependency on `reactabular-dnd` shouldn't be necessary from target application's perspective. Adding only the `kubevirt-web-ui-components` one should be enough.

Please note that:

- `patternfly-react` declares [`reactabular-table`](https://github.com/reactabular/reactabular/tree/master/packages/reactabular-table) as a regular dependency (it's used to implement PF-React [`Table`](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/src/components/Table/Table.js) component)
- `kubevirt-web-ui-components` uses [`reactabular-dnd`](https://github.com/reactabular/reactabular/tree/master/packages/reactabular-dnd) which adds drag'n'drop support to Reactabular tables (to be used with PF-React `Table` component)

However, `kubevirt-web-ui-components` package currently doesn't declare `reactabular-dnd` as its dependency, which is a problem we're going to fix (along with some other dependency issues). See [here](https://github.com/kubevirt/web-ui-components/issues/136) for more details.

So with `kubevirt-web-ui-components`, we'll try not to introduce any additional peer dependencies (on top of existing ones that come from PF-React).

Currently, your dependency tree includes `reactabular-table`. After adding `kubevirt-web-ui-components`, it will also include `reactabular-dnd`. With webpack, code for `reactabular-dnd` should be (lazy) loaded only as part of KubeVirt specific chunk (assuming the main application doesn't use `reactabular-dnd`).

Reply:
PRs for `patternfly-react` and `kubevirt-web-ui-components` projects posted, these should clean up dependencies and reduce warnings when calling `yarn install`.

@spadgett In order to eliminate unmet peer dependency warnings on `lodash` (coming mostly from `patternfly-react`'s transitive deps), I'm planning to add `"lodash": "npm:lodash-es"` [custom alias](https://yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-alias).